src/plugins/intel_cpu/src/node.h

```diff
@@ -774,9 +774,13 @@ class Node {
                                        NameFromType(getType()));
     }
 
+    DnnlScratchPadPtr getScratchPad() const {
+        return context->getScratchPad(curNumaNode);
+    }
+
     MemoryPtr getScratchPadMem(const DnnlMemoryDescPtr& desc) {
         if (!scratchpadMem || !scratchpadMem->getDesc().isCompatible(*desc)) {
-            scratchpadMem = context->getScratchPad(curNumaNode)->createScratchPadMem(desc);
+            scratchpadMem = getScratchPad()->createScratchPadMem(desc);
```

Comment:
@maxnick Maxim, could you take a look please on the changes in CPU Plugin part? Thank you in advance!